Makefile

```diff
@@ -109,9 +109,11 @@ cron/data/request.pb.go: cron/data/request.proto |  $(PROTOC)
 
 generate-docs: ## Generates docs
 generate-docs: checks/checks.md
```

Comment:
update the step 10 of https://github.com/ossf/scorecard/blob/main/checks/write.md with make generate-docs. I was not aware we had this in the Makefile myself :-)

Reply:
Done.